dotnet-roslyn/config/Audacia.CodeAnalysis.EntityFrameworkCore/Audacia.CodeAnalysis.EntityFrameworkCore.csproj

```diff
@@ -1,7 +1,10 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
     <PropertyGroup>
-        <TargetFrameworks>netstandard2.0</TargetFrameworks>
-        <Description>Code analysis packages and rulesets for apps using Entity Framework Core.</Description>
+        <TargetFramework>netstandard2.0</TargetFramework>
+        <Description>
+            Code analysis packages and rulesets for apps using Entity Framework Core.
+            This package supports workspaces operating a minimum version of C# 12 and .NET 8.
```

Comment:
Can we say "This package supports projects..." rather than "...workspaces..."? Same goes for the other .csproj change.